modules/modules.go

```diff
@@ -50,18 +52,25 @@ func (m *builder) Build(cfg config.Modules, client *http.Client) (hooks.HookRepo
 			id := fmt.Sprintf("%s.%s", vendor, moduleName)
 			if data, ok := cfg[vendor][moduleName]; ok {
 				if conf, err = json.Marshal(data); err != nil {
-					return nil, fmt.Errorf(`failed to marshal "%s" module config: %s`, id, err)
+					return nil, nil, fmt.Errorf(`failed to marshal "%s" module config: %s`, id, err)
 				}
 			}
 
 			module, err := builder(conf, client)
 			if err != nil {
-				return nil, fmt.Errorf(`failed to init "%s" module: %s`, id, err)
+				return nil, nil, fmt.Errorf(`failed to init "%s" module: %s`, id, err)
 			}
 
 			modules[id] = module
 		}
 	}
 
-	return hooks.NewHookRepository(modules)
+	coll, err := createModuleStageNamesCollection(modules)
```

Comment:
Nitpick: Could you re-name `coll` to be more descriptive

Reply:
done